hyperspy/tests/axes/test_axes_manager.py

```diff
@@ -216,3 +216,39 @@ def test_changing_scale_offset(self):
             signal_axis1.low_value, signal_axis1.high_value,
         )
         assert signal_extent == s.axes_manager.signal_extent
+
+
+class CallCounter:
```

Comment:
While this is simple, I'd rather stick to already in-built methods to do this:
```python
In [1]: from unittest import mock

In [2]: m = mock.Mock()

In [3]: m()
Out[3]: <Mock name='mock()' id='140657786956880'>

In [4]: m.call_count
Out[4]: 1
```

Reply:
Yes, indeed!